cmd/jujud/agent/model/manifolds_test.go

```diff
@@ -166,3 +167,442 @@ func (s *ManifoldsSuite) TestClockWrapper(c *gc.C) {
 }
 
 type fakeClock struct{ clock.Clock }
+
+func (s *ManifoldsSuite) assertManifoldsDependencies(c *gc.C, manifolds dependency.Manifolds, expected map[string][]string) {
+	dependencies := make(map[string][]string, len(manifolds))
+	manifoldNames := set.NewStrings()
+
+	for name, manifold := range manifolds {
+		manifoldNames.Add(name)
+		dependencies[name] = []string{}
+
+		if len(manifold.Inputs) == 0 {
+			continue
+		}
+		manifoldDependencies := manifolds.ManifoldDependencies(name, manifolds[name])
+		for _, input := range manifoldDependencies.SortedValues() {
```

Comment:
`set.SortedValues()` returns a slice, so can't you just set that as the value in the map?

```
dependencies[name] = manifoldDependencies.SortedValues()
```